google-shopping.js

```diff
@@ -1,16 +1,82 @@
 // Write your solutions below
 const jsonfile = require('jsonfile');
-
-const file = 'products.json'
+const file = 'products.json';
 
 jsonfile.readFile(file, function(err, obj) {
-  console.dir(obj)
+  //Getting all relevant data
+  var items = obj.items;
+  //making it shorter
+  var questionOne = items.filter(x => {
+    return x.kind === "shopping#product";
+  })
+
+  function questionTwo() {
+    var results = items.filter(x => {
+      return x.product.inventories[0].availability == "backorder"
+    });
+    // console.log(results[0].kind);
+
+    var iterated = results.forEach(function(log){
+      console.log(typeof log['product'].title);
+      return log['product'];
+      })
+  }
 
-  var resultFile = 'result.json'
-  var result = {name: 'JP'}
+  function questionThree() {
+    for (i = 0; i < items.length; i++) {
+      if (items[i].product.images.length >= 2)
+        return {
```

Comment:
Whenever there is a return statement it breaks the code block.
So its best to put the return statement towards the end of the function block and out of the for loop.